<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NotepadXWindow</class>
 <widget class="QMainWindow" name="NotepadXWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>964</width>
    <height>737</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>NotepadX</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QPlainTextEdit" name="markdownEdit"/>
      </item>
      <item row="0" column="1">
       <widget class="QTextBrowser" name="markdownPreview">
        <property name="minimumSize">
         <size>
          <width>400</width>
          <height>200</height>
         </size>
        </property>
        <property name="contextMenuPolicy">
         <enum>Qt::NoContextMenu</enum>
        </property>
        <property name="acceptDrops">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>964</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>文件(&amp;F)</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSaveAs"/>
    <addaction name="separator"/>
    <addaction name="actionPrint"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>编辑(&amp;E)</string>
    </property>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
    <addaction name="separator"/>
    <addaction name="actionCut"/>
    <addaction name="actionCopy"/>
    <addaction name="actionPaste"/>
    <addaction name="actionDelete"/>
    <addaction name="separator"/>
    <addaction name="actionAll"/>
    <addaction name="actionFind"/>
   </widget>
   <widget class="QMenu" name="menuFormt">
    <property name="title">
     <string>格式(&amp;O)</string>
    </property>
    <addaction name="actionFont"/>
    <addaction name="actionBold"/>
    <addaction name="actionItalic"/>
    <addaction name="actionUnderline"/>
    <addaction name="actionStrikeout"/>
    <addaction name="actionColor"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>查看(&amp;V)</string>
    </property>
    <widget class="QMenu" name="menu_Z">
     <property name="title">
      <string>缩放(&amp;Z)</string>
     </property>
     <addaction name="actionEnlarge"/>
     <addaction name="actionLessen"/>
     <addaction name="actionDefault"/>
    </widget>
    <addaction name="menu_Z"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuFormt"/>
   <addaction name="menuView"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="allowedAreas">
    <set>Qt::TopToolBarArea</set>
   </property>
   <property name="iconSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonIconOnly</enum>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew"/>
   <addaction name="actionOpen"/>
   <addaction name="actionSave"/>
   <addaction name="actionPrint"/>
   <addaction name="separator"/>
   <addaction name="actionCopy"/>
   <addaction name="actionCut"/>
   <addaction name="actionPaste"/>
   <addaction name="separator"/>
   <addaction name="actionUndo"/>
   <addaction name="actionRedo"/>
   <addaction name="separator"/>
   <addaction name="actionBold"/>
   <addaction name="actionItalic"/>
   <addaction name="actionUnderline"/>
   <addaction name="actionStrikeout"/>
   <addaction name="actionColor"/>
   <addaction name="actionFont"/>
   <addaction name="separator"/>
   <addaction name="actionEnlarge"/>
   <addaction name="actionLessen"/>
   <addaction name="separator"/>
   <addaction name="actionSynchronize"/>
   <addaction name="separator"/>
   <addaction name="actionAbout"/>
  </widget>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/ui/Resources/new.png</normaloff>:/ui/Resources/new.png</iconset>
   </property>
   <property name="text">
    <string>新建(&amp;N)</string>
   </property>
   <property name="toolTip">
    <string>新建(Ctrl+N)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/ui/Resources/open.png</normaloff>:/ui/Resources/open.png</iconset>
   </property>
   <property name="text">
    <string>打开(&amp;O)</string>
   </property>
   <property name="toolTip">
    <string>打开(Ctrl+O)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/ui/Resources/save.png</normaloff>:/ui/Resources/save.png</iconset>
   </property>
   <property name="text">
    <string>保存(&amp;S)</string>
   </property>
   <property name="toolTip">
    <string>保存(Ctrl+S)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="text">
    <string>保存为(&amp;A)</string>
   </property>
   <property name="toolTip">
    <string>保存为...</string>
   </property>
   <property name="shortcut">
    <string>F12</string>
   </property>
  </action>
  <action name="actionPrint">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/ui/Resources/print.png</normaloff>:/ui/Resources/print.png</iconset>
   </property>
   <property name="text">
    <string>打印(&amp;P)</string>
   </property>
   <property name="toolTip">
    <string>打印(Ctrl+P)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>退出(&amp;X)</string>
   </property>
   <property name="toolTip">
    <string>退出</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/ui/Resources/cut.png</normaloff>:/ui/Resources/cut.png</iconset>
   </property>
   <property name="text">
    <string>剪切(&amp;T)</string>
   </property>
   <property name="toolTip">
    <string>剪切(Ctrl+X)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/ui/Resources/copy.png</normaloff>:/ui/Resources/copy.png</iconset>
   </property>
   <property name="text">
    <string>复制(&amp;C)</string>
   </property>
   <property name="toolTip">
    <string>复制(Ctrl+C)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/ui/Resources/paste.png</normaloff>:/ui/Resources/paste.png</iconset>
   </property>
   <property name="text">
    <string>粘贴(&amp;P)</string>
   </property>
   <property name="toolTip">
    <string>粘贴(Ctrl+P)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="text">
    <string>删除(&amp;L)</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
  <action name="actionFind">
   <property name="text">
    <string>查找(&amp;F)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="actionAll">
   <property name="text">
    <string>全选(&amp;A)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="actionReplace">
   <property name="text">
    <string>替换(&amp;R)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+H</string>
   </property>
  </action>
  <action name="actionGoto">
   <property name="text">
    <string>转到(&amp;G)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+G</string>
   </property>
  </action>
  <action name="actionBold">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/ui/Resources/bold.png</normaloff>:/ui/Resources/bold.png</iconset>
   </property>
   <property name="text">
    <string>加粗(&amp;B)</string>
   </property>
   <property name="toolTip">
    <string>加粗(Ctrl+B)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+B</string>
   </property>
  </action>
  <action name="actionItalic">
   <property name="icon">
    <iconset>
     <normalon>:/ui/Resources/italic.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>斜体(&amp;I)</string>
   </property>
   <property name="toolTip">
    <string>斜体(Ctrl+I)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="actionStatus">
   <property name="text">
    <string>状态栏(&amp;S)</string>
   </property>
  </action>
  <action name="actionEnlarge">
   <property name="icon">
    <iconset>
     <normalon>:/ui/Resources/enlarge.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>放大</string>
   </property>
   <property name="toolTip">
    <string>放大(Ctrl++)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl++</string>
   </property>
  </action>
  <action name="actionLessen">
   <property name="icon">
    <iconset>
     <normalon>:/ui/Resources/lessen.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>缩小</string>
   </property>
   <property name="toolTip">
    <string>缩小(Ctrl+-)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+-</string>
   </property>
  </action>
  <action name="actionDefault">
   <property name="text">
    <string>恢复默认缩放</string>
   </property>
   <property name="toolTip">
    <string>恢复默认缩放(Ctrl+0)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+0</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/ui/Resources/about.png</normaloff>:/ui/Resources/about.png</iconset>
   </property>
   <property name="text">
    <string>关于NotepadX</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="icon">
    <iconset>
     <normalon>:/ui/Resources/undo.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>撤销(&amp;Z)</string>
   </property>
   <property name="toolTip">
    <string>撤销(Ctrl+Z)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="icon">
    <iconset>
     <normalon>:/ui/Resources/redo.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>重做(&amp;Y)</string>
   </property>
   <property name="toolTip">
    <string>重做(Ctrl+Y)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Y</string>
   </property>
  </action>
  <action name="actionUnderline">
   <property name="icon">
    <iconset>
     <normalon>:/ui/Resources/underline.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>下划线(&amp;U)</string>
   </property>
   <property name="toolTip">
    <string>下划线(Ctrl+U)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+U</string>
   </property>
  </action>
  <action name="actionFont">
   <property name="icon">
    <iconset>
     <normalon>:/ui/Resources/font.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>字体(&amp;F)</string>
   </property>
   <property name="toolTip">
    <string>字体(Ctrl+Shift+F)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+F</string>
   </property>
  </action>
  <action name="actionStrikeout">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/ui/Resources/strikeout.png</normaloff>:/ui/Resources/strikeout.png</iconset>
   </property>
   <property name="text">
    <string>删除线</string>
   </property>
   <property name="toolTip">
    <string>删除线(Ctrl+Shift+D)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+D</string>
   </property>
  </action>
  <action name="actionColor">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/ui/Resources/color.png</normaloff>:/ui/Resources/color.png</iconset>
   </property>
   <property name="text">
    <string>颜色</string>
   </property>
   <property name="toolTip">
    <string>颜色(Ctrl+Shift+C)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+C</string>
   </property>
  </action>
  <action name="actionSynchronize">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/ui/Resources/nosynchronize.png</normaloff>
     <normalon>:/ui/Resources/synchronize.png</normalon>:/ui/Resources/nosynchronize.png</iconset>
   </property>
   <property name="text">
    <string>滚动同步</string>
   </property>
   <property name="toolTip">
    <string>滚动同步</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionAll</sender>
   <signal>triggered()</signal>
   <receiver>markdownEdit</receiver>
   <slot>selectAll()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>260</x>
     <y>353</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionCopy</sender>
   <signal>triggered()</signal>
   <receiver>markdownEdit</receiver>
   <slot>copy()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>260</x>
     <y>353</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionCut</sender>
   <signal>triggered()</signal>
   <receiver>markdownEdit</receiver>
   <slot>cut()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>260</x>
     <y>353</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionPaste</sender>
   <signal>triggered()</signal>
   <receiver>markdownEdit</receiver>
   <slot>paste()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>260</x>
     <y>353</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionNew</sender>
   <signal>triggered()</signal>
   <receiver>markdownEdit</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>260</x>
     <y>353</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionRedo</sender>
   <signal>triggered()</signal>
   <receiver>markdownEdit</receiver>
   <slot>redo()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>260</x>
     <y>353</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionUndo</sender>
   <signal>triggered()</signal>
   <receiver>markdownEdit</receiver>
   <slot>undo()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>260</x>
     <y>353</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>NotepadXWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>526</x>
     <y>339</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionLessen</sender>
   <signal>triggered()</signal>
   <receiver>markdownPreview</receiver>
   <slot>zoomOut()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>788</x>
     <y>353</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionEnlarge</sender>
   <signal>triggered()</signal>
   <receiver>markdownPreview</receiver>
   <slot>zoomIn()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>788</x>
     <y>353</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
